osu.Game.Rulesets.Osu/Objects/Slider.cs

```diff
@@ -81,6 +81,9 @@ public override Vector2 Position
 
         public List<IList<HitSampleInfo>> NodeSamples { get; set; } = new List<IList<HitSampleInfo>>();
 
+        [JsonIgnore]
+        public IList<HitSampleInfo> OriginalSamples { get; private set; }
```

Comment:
This is not updating on changes to `Samples` / `SamplesBindable`, i.e. in editor, adding or removing the whisper sample from the slider will not update the drawable object.

Reply:
Oh I thought that such changes would call `ApplyDefaults`? Will need to check but I’m not sure it’s correct right now either.

Reply:
Fixed with the newest changes.